### PR TITLE
Hubble Space Telescope updates + Bugfixes

### PR DESCRIPTION
**Pull request by MeuhV1**
Corrected Hubble Space Telescope pedia text
Hubble Space Telescope has space environment in the pedia

### CODE REVIEW

**Comment by MeuhV1** [comment]
Commit 2:
Bugfix in RFCUtils
Exported ChangeBuildingYieldChange to python. (Required for the Space Elevator and probably some other wonders/projects as well)
Spaceship screen doesn't automatically open when a space project is finished.

File: CvGameCoreDLL/CvCity.cpp [hunk 1]
```diff
@@ -14415,38 +14415,39 @@ void CvCity::popOrder(int iNum, bool bFinish, bool bChoose)
 					{
 						CvPopupInfo* pInfo = NULL;
 
-						if (GC.getGameINLINE().isNetworkMultiPlayer())
```
Comment (dguenms) [inline CvGameCoreDLL/CvCity.cpp]: Can you use multiline comments? Like this:
```
/* This is
a multiline
comment*/
```

File: CvGameCoreDLL/CvCity.cpp [hunk 2]
```diff
@@ -14415,38 +14415,39 @@ void CvCity::popOrder(int iNum, bool bFinish, bool bChoose)
...
-						if(!GC.getGameINLINE().isNetworkMultiPlayer())
-						{
-							CvPopupInfo* pInfo = new CvPopupInfo(BUTTONPOPUP_PYTHON_SCREEN, eCreateProject);
-							pInfo->setText(L"showSpaceShip");
-							gDLL->getInterfaceIFace()->addPopup(pInfo, getOwnerINLINE());
-							needsArtType = false;
-						}
-					}
-				}
+				// else
```
Comment (dguenms) [inline CvGameCoreDLL/CvCity.cpp]: Same here.

**Review by dguenms — APPROVED** [review]
Ready to merge after rebase.

**Comment by MeuhV1** [comment]
Done.

**Review by dguenms — APPROVED** [review]

**Pull request merged**